.github/workflows/deploy.yml

```
@@ -0,0 +1,75 @@
+name:  Spring Boot & Gradle CI/CD
```

YAMLlint (1.37.1)

[error] too many spaces after colon

(colons)